pixeltable/func/together_function.py

```diff
@@ -0,0 +1,77 @@
+from typing import Optional, Any, Dict, List, Tuple
+import logging
+import inspect
+import dataclasses
+
+import numpy as np
+import jmespath
+
+from .signature import Signature, Parameter
+from .function_md import FunctionMd
+from .external_function import ExternalFunction
+import pixeltable.env as env
+import pixeltable.type_system as ts
+import together
```

Comment:
let's move this into invoke(), to avoid an import without the package being installed

Reply:
?

Reply:
Good call (sorry missed this comment earlier)